packages/web/src/pages/profile-page/components/desktop/ProfileWrapping.js

```diff
@@ -349,7 +350,10 @@ const ProfileWrapping = props => {
                 onChange={props.onUpdateName}
                 userId={props.userId}
               />
-              <h2 className={styles.handle}>{props.handle}</h2>
+              <div className={styles.handleWrapper}>
+                <h2 className={styles.handle}>{props.handle}</h2>
+                {props.doesFollowCurrentUser && <FollowsYouBadge />}
```

Comment:
also, we have a new preference for using terneries instead of && for conditional rendering: https://www.notion.so/audiusproject/Client-Javascript-Styleguide-4f34640139fc4a809c3765131bc597ff#f1c89a4e0843430bbf98f0ce53dbcf14